src/main/java/sopt35/linkareer/api/dto/response/OfficialListResponse.java

```diff
@@ -0,0 +1,23 @@
+package sopt35.linkareer.api.dto.response;
+
+import sopt35.linkareer.domain.official.application.dto.response.OfficialDto;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/*
+List<Official>를 받아 List<OfficialResponse>로 변환
+ */
+public record OfficialListResponse(List<OfficialResponse> officialList) {
+
+    // official 리스트를 받아 OfficialResponse 리스트로 변환
+    public static OfficialListResponse from(List<OfficialDto> officialList) {
+        List<OfficialResponse> OfficialResponse = officialList.stream()
+                .map(official -> new OfficialResponse(
```

Comment:
```suggestion
                .map(official -> OfficialResponse.of(official)
```

요러케 하면 조금 더 가독성이 좋아질 거 같아요! 내부에서 파라미터에 넣어서 `팩토리 메서드 패턴`으로 만들어주면 좋을 거 같아요! 

출처 : https://jusungpark.tistory.com/14 해당 `팩토리 메서드 패턴`

Reply:
넵 반영할게요!